src/pages/WorkspaceSwitcherPage/index.tsx

```diff
@@ -170,26 +170,22 @@ function WorkspaceSwitcherPage() {
             testID={WorkspaceSwitcherPage.displayName}
             includeSafeAreaPaddingBottom={false}
         >
-            {({didScreenTransitionEnd}) => (
-                <>
-                    <HeaderWithBackButton
-                        title={translate('workspace.switcher.headerTitle')}
-                        onBackButtonPress={Navigation.goBack}
-                    />
-                    <SelectionList<WorkspaceListItem>
-                        ListItem={UserListItem}
-                        sections={didScreenTransitionEnd ? sections : CONST.EMPTY_ARRAY}
-                        onSelectRow={selectPolicy}
-                        textInputLabel={usersWorkspaces.length >= CONST.WORKSPACE_SWITCHER.MINIMUM_WORKSPACES_TO_SHOW_SEARCH ? translate('common.search') : undefined}
-                        textInputValue={searchTerm}
-                        onChangeText={setSearchTerm}
-                        headerMessage={headerMessage}
-                        listFooterContent={shouldShowCreateWorkspace ? WorkspaceCardCreateAWorkspaceInstance : null}
-                        initiallyFocusedOptionKey={activeWorkspaceID ?? CONST.WORKSPACE_SWITCHER.NAME}
-                        showLoadingPlaceholder
-                    />
-                </>
-            )}
+            <HeaderWithBackButton
+                title={translate('workspace.switcher.headerTitle')}
+                onBackButtonPress={Navigation.goBack}
+            />
+            <SelectionList<WorkspaceListItem>
+                ListItem={UserListItem}
+                sections={sections}
```

Comment:
do we default the sections to [] if not yet defined?

Reply:
@mountiny I think we don't need the default value here because we always have this option 

<img width="377" alt="Screenshot 2024-04-19 at 17 43 01" src="https://github.com/Expensify/App/assets/141406735/1c2bb714-b2d9-4641-9ca6-eca1ef707784">